src/Mod/TechDraw/Gui/CommandExtensionPack.cpp

```diff
@@ -777,8 +777,8 @@ CmdTechDrawExtensionChangeLineAttributes::CmdTechDrawExtensionChangeLineAttribut
 {
     sAppModule = "TechDraw";
     sGroup = QT_TR_NOOP("TechDraw");
-    sMenuText = QT_TR_NOOP("Change Line Attributes");
-    sToolTipText = QT_TR_NOOP("Change the attributes of cosmetic lines and centerlines:<br>\
+    sMenuText = QT_TR_NOOP("Change line attributes");
+    sToolTipText = QT_TR_NOOP("Changes the attributes of the selected cosmetic lines and centerlines:<br>\
```

Comment:
```suggestion
    sToolTipText = QT_TR_NOOP("Changes the selected cosmetic lines and centerlines to the specified attributes")
```
then remove next 3 lines